src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -222,8 +222,6 @@ public sealed partial class JsonSerializerOptions
         public System.Text.Json.Serialization.JsonIgnoreCondition DefaultIgnoreCondition { get { throw null; } set { } }
         public System.Text.Json.JsonNamingPolicy? DictionaryKeyPolicy { get { throw null; } set { } }
         public System.Text.Encodings.Web.JavaScriptEncoder? Encoder { get { throw null; } set { } }
-        [System.ObsoleteAttribute("To ignore null values when serializing, set DefaultIgnoreCondition to JsonIgnoreCondition.WhenWritingNull.", false)]
```

Comment:
Is this intentional?

Reply:
It was stripped by the generator (`t:GenerateReferenceAssemblySource `). Will add it back. cc @safern FYI.

Reply:
Yeah, `ObsoleteAttributes` are included in the list to "ignore" for APICompat intentionally.